services/mongo/tokenService.go

```diff
@@ -0,0 +1,66 @@
+package mongo
+
+import (
+	"context"
+	"github.com/unicsmcr/hs_auth/entities"
+	"github.com/unicsmcr/hs_auth/environment"
+	"github.com/unicsmcr/hs_auth/repositories"
+	"github.com/unicsmcr/hs_auth/services"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.uber.org/zap"
+)
+
+type mongoTokenService struct {
+	logger          *zap.Logger
+	env             *environment.Env
+	tokenRepository *repositories.TokenRepository
+}
+
+// NewMongoTokenService creates a new TokenService that uses MongoDB as the storage technology
+func NewMongoTokenService(logger *zap.Logger, env *environment.Env, tokenRepository *repositories.TokenRepository) services.TokenService {
+	return &mongoTokenService{
+		logger:          logger,
+		env:             env,
+		tokenRepository: tokenRepository,
+	}
+}
+
+func (s *mongoTokenService) AddServiceToken(ctx context.Context, tokenId primitive.ObjectID, creatorID, jwt string) (*entities.Token, error) {
```

Comment:
```suggestion
func (s *mongoTokenService) CreateServiceToken(ctx context.Context, tokenId primitive.ObjectID, creatorID, jwt string) (*entities.Token, error) {
```

Reply:
Yeah, I guess that makes more sense in the context of the service 👍

Reply:
Also, we should maybe follow the pattern used in `UserService` and `TeamService` and create the id within `TokenService` rather than providing it to `CreateServiceToken`, it'll be easier to control what kind of ids are used.